### PR TITLE
I add a translation of the readme into Spanish

### PR DESCRIPTION
**Pull request by FranciscoImanolSuarez**
I add translation for "Working with data"

### CODE REVIEW

File: 2-Working-With-Data/translations/README.es.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Trabajando con datos
+
+![Amor por los datos](images/data-love.jpg)
+> Fotografia de <a href="https://unsplash.com/@swimstaralex?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Alexander Sinn</a> on <a href="https://unsplash.com/s/photos/data?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>
+  
+En estas lecciones, aprenderás algunas de las formas para manejar datos, también aprenderás a manipularlos y usarlos en aplicaciones. Aprendera sobre bases de datos relacionales y no relacionales así como también almacenar datos en estas. Veremos los fundamentos de Python para poder administrar datos y extraerlos.
+
+### Temas
+
+1. [Bases de datos relacionales](05-relational-databases/README.md)
```
Comment (jlooper) [inline 2-Working-With-Data/translations/README.es.md:10]: hi, can you ensure that these links are valid, probably you need relative links back to the English versions

**Review by jlooper — CHANGES_REQUESTED** [review]

File: 2-Working-With-Data/translations/README.es.md (hunk 1 above)
Comment (FranciscoImanolSuarez) [reply 2-Working-With-Data/translations/README.es.md:10]: Hello! Is it ok if I first translate "05-relational-bases/README.md" and then link it here?

Comment (paladique) [reply 2-Working-With-Data/translations/README.es.md:10]: @FranciscoImanolSuarez that's fine if you'd like to, but let's get this one merged first. I suggest updating the links first (links to lessons 5,6,7,8 will need to be updated in this readme) and then submit your lesson translation in a new PR.

Comment (FranciscoImanolSuarez) [reply 2-Working-With-Data/translations/README.es.md:10]: Perfect @paladique , unfortunately I can't understand what the error of the links is.

Sorry for the delay in responding, from today I will dedicate more time to this.

Comment (paladique) [reply 2-Working-With-Data/translations/README.es.md:10]: No problem, these can take some time and we appreciate the time you've spent on it so far. The English versions can be accessed outside the `translations` directory like this:

 `[Bases de datos relacionales](../05-relational-databases/README.md)`

Let us know if you have any other questions

Comment (FranciscoImanolSuarez) [reply 2-Working-With-Data/translations/README.es.md]: Problems solved

**Review by paladique — APPROVED** [review]

**Comment by paladique** [comment]
@FranciscoImanolSuarez Can you resolve the conflict in your branch before I merge this? Thanks!

**Comment by FranciscoImanolSuarez** [comment]
I will close this pull request and submit it again.

**Comment by FranciscoImanolSuarez** [comment]
New PR #333

**Pull request closed without merging**